### PR TITLE
Grap 55

### PR DESCRIPTION
**Pull request by ppmarkek**

### CODE REVIEW

**Review by svyatoslavkk — APPROVED** [review]

File: src/components/templates/Header/Header.tsx [hunk 1]
```diff
@@ -33,15 +33,18 @@ const Header = () => {
   return (
     <Wrapper container isSticky={isSticky}>
       <Grid item xs={6}>
-        <StyledLink width="150px" to={'/'}>
+        <StyledLink isSticky={isSticky} width="150px" to={'/'}>
           Welcome page
```
Comment (1LisaW) [inline src/components/templates/Header/Header.tsx:37]: Could you please add here i18n? Text should be dependand on current language.

File: src/components/templates/Header/Header.tsx [hunk 2]
```diff
@@ -33,15 +33,18 @@ const Header = () => {
...
         </StyledLink>
       </Grid>
 
       <Grid container item xs={6} justifyContent={'flex-end'} gap={'20px'}>
         <StyledFormControl>
-          <InputLabel id="demo-simple-select-label">Language</InputLabel>
```
Comment (1LisaW) [inline src/components/templates/Header/Header.tsx:43]: Could you please add here i18n? Text should be dependand on current language.

File: src/components/templates/Header/Header.tsx [hunk 3]
```diff
@@ -33,15 +33,18 @@ const Header = () => {
...
-          <Select
+          <StyledInputLabel isSticky={isSticky} id="demo-simple-select-label">
+            Language
```
Comment (1LisaW) [inline src/components/templates/Header/Header.tsx:44]: Could you please add here i18n? Text should be dependand on current language.

File: src/components/templates/Header/Header.tsx [hunk 4]
```diff
@@ -50,9 +53,9 @@ const Header = () => {
           >
             <MenuItem value={Langs.ru}>RU</MenuItem>
             <MenuItem value={Langs.en}>EN</MenuItem>
-          </Select>
+          </StyledSelect>
         </StyledFormControl>
-        <StyledLink width="100px" to={'/signOut'}>
+        <StyledLink isSticky={isSticky} width="100px" to={'/signOut'}>
           Sign Out
```
Comment (1LisaW) [inline src/components/templates/Header/Header.tsx:59]: Could you please add here i18n? Text should be dependand on current language.

**Review by 1LisaW — APPROVED** [review]

**Pull request merged**